src/RA_Core.cpp

```diff
@@ -397,8 +397,9 @@ API void CCONV _RA_OnReset()
 
 API void CCONV _RA_InstallMemoryBank(int nBankID, void* pReader, void* pWriter, int nBankSize)
 {
-    g_MemManager.AddMemoryBank(static_cast<size_t>(nBankID), (_RAMByteReadFn*)pReader, (_RAMByteWriteFn*)pWriter, static_cast<size_t>(nBankSize));
-    g_MemoryDialog.AddBank(nBankID);
+    g_MemManager.AddMemoryBank(ra::to_unsigned(nBankID), (_RAMByteReadFn*)pReader, (_RAMByteWriteFn*)pWriter,
```

Comment:
Oddly enough, it didn't care about C-Style casts here but it did care about the `static_cast`s.